docs/index.rst

```diff
@@ -106,7 +106,7 @@ The only thing you have to do is provide a DSN, either by registering an instanc
 
     protected override void ApplicationStartup(TinyIoCContainer container, IPipelines pipelines)
     {
-        container.Register(new Dsn("http://public:secret@example.com/project-id"));
+        container.Register(new Dsn("http://public@example.com/project-id"));
```

Comment:
*nit*: You could use `___PUBLIC_DSN___` here to show an interactive DSN chooser in docs and the setup wizard:

```csharp
container.Register(new Dsn("___PUBLIC_DSN___"));
```

Reply:
Done!